src/vumi2/applications/junebug_message_api/junebug_state_cache.py

```diff
@@ -48,6 +48,7 @@ async def delete_inbound(self, message_id: str) -> None:
 @define
 class MemoryJunebugStateCacheConfig:
     timeout: float = 60 * 60 * 24
+    store_event_info: bool = True
```

Comment:
Do we also want this for inbound messages if we know we're never going to send replies? Or maybe just a `noop` cache that doesn't store anything?

Reply:
Something similar for inbound messages seems reasonable, but let's do it in a new PR.